client/src/components/buttonSelector/buttonSelector/ButtonSelector.jsx

```diff
@@ -11,19 +11,25 @@ const ButtonSelector = ({
   style,
   classNameButton,
   classNameSelector,
+  scroll,
 }) => {
   function setActiveButton(index) {
     setActiveIndex(index);
   }
 
+  const [scrollStatus, setScrollStatus] = useState(true);
+
+  useEffect(() => {
+    setScrollStatus(scroll);
+  }, [scroll]);
+
   const buttonItems = buttonList.map((item, index) => (
     <ButtonOutlined
       key={index}
       onClick={() => setActiveButton(index)}
       label={item.name}
       style={{
         backgroundColor: `${item.buttonColor}`,
-        flex: 1,
```

Comment:
Again, don't remove styles that can have an impact on other pages using this component
Maybe make this `unset` when setting the scroll prop

Reply:
Fixed with boolean 'scroll' prop